packages/katello/katello/katello-debug.sh

```diff
@@ -69,15 +69,13 @@ add_cmd "sudo -u pulp PULP_SETTINGS='/etc/pulp/settings.py' DJANGO_SETTINGS_MODU
 # Qpidd (*)
 if [ $NOGENERIC -eq 0 ]; then
   add_files /etc/qpid/*
-  add_files /etc/qpidd.conf
   add_files /etc/qpid-dispatch/qdrouterd.conf
-  add_files /var/log/qdrouterd/qdrouterd.log
 fi
 
-if [ -f /etc/pki/katello/qpid_client_striped.crt ]; then
-  add_cmd "qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671" "qpid-stat-q"
-  add_cmd "qpid-stat -u --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671" "qpid-stat-u"
-  add_cmd "qpid-stat -c --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671" "qpid-stat-c"
+if [ -f /etc/pki/katello/qpid_router_client.crt ]; then
+  add_cmd "qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_router_client.crt --ssl-key=/etc/pki/katello/qpid_router_client.key -b amqps://localhost:5671" "qpid-stat-q"
+  add_cmd "qpid-stat -u --ssl-certificate=/etc/pki/katello/qpid_router_client.crt --ssl-key=/etc/pki/katello/qpid_router_client.key -b amqps://localhost:5671" "qpid-stat-u"
+  add_cmd "qpid-stat -c --ssl-certificate=/etc/pki/katello/qpid_router_client.crt --ssl-key=/etc/pki/katello/qpid_router_client.key -b amqps://localhost:5671" "qpid-stat-c"
```

Comment:
To allow these to work on EL8 as well, we need them to look like:

```suggestion
  add_cmd "qpid-stat -c --ssl-certificate=/etc/pki/katello/qpid_router_client.crt --ssl-key=/etc/pki/katello/qpid_router_client.key --sasl-mechanism=EXTERNAL -b amqps://$HOSTNAME@localhost:5671" "qpid-stat-c"
```